nasa/harmony-regression-tests
# DAS-1601 - Remove PNG and GIF tests from HGA suite.

## Description

This PR removes all tests from the HGA regression tests suite that ask for PNG or GIF outputs. This is in anticipation of a PR against Harmony to remove those output format options from the HGA service chain (and corresponding UMM-S changes).

The bottom line here: we were going to do this when a net2cog and HyBIG chain existed. But almost no-one is using HGA for these operations (and the GIF requests often fail), so let's remove the option for requesting these outputs now. Once that's done and no-one is impacted, we can cut the code from HGA itself and _finally_ complete DAS-1601.

## Jira Issue ID

DAS-1601 (preliminary work).

## Local Test Steps

* Pull this branch.
* Build the HGA test image (`cd test && make hga-image`).
* Run the tests against UAT or SIT (they should pass):

```
export EDL_USER=<your EDL user>
export EDL_PASSWORD=<your EDL password>
export HARMONY_HOST_URL=https://harmony.uat.earthdata.nasa.gov
./run-notebooks.sh hga
```

## PR Acceptance Checklist
* ~~Acceptance criteria met~~ (This is enabling work for DAS-1601)
* ~~Documentation updated (if needed)~~
